data/gegno/gegno power.txt

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) 2022 by Saugia
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+outfit "R01 Skirmish Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R01 Skirmish Batteries"
+	cost 17500
+	mass 12
+	thumbnail outfit/gbatterytiny
+	"energy capacity" 1121
+	"outfit space" -12
+	description "Skirmish batteries are very common amongst ships across Gegno space. They can be either a perfectly convenient storage of battery power to offset the load of larger Gegno generators, or a nice addition to add on top of them where more energy would be needed."
```

Comment:
Amongst is discouraged in American English. Also what is the second sentence saying?
```suggestion
	description "Skirmish batteries are very common among ships across Gegno space. They can be either a perfectly convenient storage of battery power to offset the load of larger Gegno generators, or a nice addition to add on top of them where more energy would be needed."
```

Reply:
I believe the idea was that the batteries can be uses instead of larger generators or in addition to them if desired. I'll think of a way to reword this.